.ci/Jenkinsfile

```
@@ -136,6 +136,28 @@ pipeline {
             }
           }
         }
+        stage('Build Docs') {
+          options { skipDefaultCheckout() }
+          when {
+            beforeAgent true
+            anyOf {
+              expression { return env.FORCE_SKIP_GIT_CHECKS == "true" }
+              expression { return env.SKIP_TESTS == "false" }
+            }
+          }
+          steps {
+            deleteDir()
+            unstash 'source'
+            dir("${BASE_DIR}/e2e") {
+              sh(label: 'Build docs', script: 'make build-docs')
+            }
+            post {
+              always {
+                archiveArtifacts allowEmptyArchive: true, artifacts: "${BASE_DIR}/e2e/docs/**"
```

nit: cosmetic change, if it helps to access to the URL with a few less folders

```suggestion
                dir("${BASE_DIR}") {
                  archiveArtifacts allowEmptyArchive: true, artifacts: "e2e/docs/**"
                }
```